akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -411,8 +427,36 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
 
   private def closeScheduler(): Unit = scheduler match {
     case x: Closeable ⇒ x.close()
-    case _            ⇒
+    case _            ⇒ // ignore, this is fine
+  }
+
+  // ========================================================
+  // ================== OLDEST MAINTANANCE ==================
+  // ========================================================
+
+  /** INTERNAL API: Returns state.members sorted by Member.ageOrdering (from oldest to youngest) */
+  @InternalApi private[cluster] def membersByAge(): immutable.SortedSet[Member] =
+    SortedSet.empty(Member.ageOrdering) ++ state.members.toList // FIXME
+
+  /** INTERNAL API */
+  @InternalApi
+  private[cluster] def isAtLeastNthOldestAddress(address: UniqueAddress, memberFilter: Member ⇒ Boolean, n: Int): Boolean = {
```

Comment:
this is used to make decisions if "I'm nth oldest, so I should be monitoring other nodes"